RecoTracker/LST/plugins/LSTOutputConverter.cc

```diff
@@ -163,7 +150,51 @@ void LSTOutputConverter::produce(edm::StreamID, edm::Event& iEvent, const edm::E
       }
     });
 
-    output.emplace_back(TrackCandidate(recHits,s,st));
+    if (lstTC_seedIdx[i] == -1) { // T5
+      using Hit = SeedingHitSet::ConstRecHitPointer;
```

Comment:
as discussed Aug 8, when I was updating this part of the code, the idea was to make up a new `TrajectorySeed` object only if none was already known.